jsu.js

```diff
@@ -230,7 +230,13 @@ if (shouldBeDefined('httpRequest')) {
         const urlParams = [];
         let field;
         for (field in params) {
-            urlParams.push(encodeURIComponent(field) + '=' + encodeURIComponent(params[field]));
+            if (params[field] instanceof Array) {
+                for (const data of params[field]) {
```

Comment:
The use of `data` can be confusing with `args.data`, so I think that the variable should be called `param`.